### PR TITLE
local development updates

### PR DESCRIPTION
**Pull request by jjw24**
add some local development updates to make it easier to jump in

### CODE REVIEW

**Comment by jjw24** [comment]
this zip command, what is the package you are using?
https://github.com/Flow-Launcher/Flow.Launcher.Plugin.PythonTemplate/blob/4d71e53d26f3f54f83e256610ea13d43f865fced/Flow.Launcher.Plugin.PythonTemplate/commands.py#L134

I my windows doesnt have it, can we switch to a common one used on Windows like tar (available on win 10 machines)?

File: Flow.Launcher.Plugin.PythonTemplate/commands.py [hunk 1]
```diff
@@ -72,17 +101,92 @@ def gen_plugin_info():
...
+    os.system("rm -fr build/")
+    click.echo("Done.")
+
+
+@clean.command()
+def clean_pyc():
+    "Remove Python file artifacts"
 
-    click.echo('Done.')
+    os.system(f"find {basedir} -name '*.pyc' -exec rm -f {{}} +")
```
Comment (Zeroto521) [inline Flow.Launcher.Plugin.PythonTemplate/commands.py:215]: Actually, the `find` command also doesn't exist for window.
I run it in ps env with [choco](https://chocolatey.org/) to install these commands.

But those (ex. zip, find) commands will run at CI Linux env, so don't worry about it.

**Comment by Zeroto521** [comment]
https://github.com/Zeroto521/PythonTemplate

basic done, demo seee.

File: Flow.Launcher.Plugin.PythonTemplate/commands.py [hunk 2]
```diff
@@ -72,17 +101,92 @@ def gen_plugin_info():
...
+        ".git/*",
+        ".vscode/*",
+        ".history/*",
+        "*/__pycache__/*",
+        "build/*",
+        # file
+        ".gitignore",
+        ".gitattributes",
+    ]
+    os.system(f"zip -r {zip_path} . -x {' '.join(ignore_list)}")
```
Comment (Zeroto521) [inline Flow.Launcher.Plugin.PythonTemplate/commands.py]: `zip` command seems couldn't ignore folder well.

zip from here https://github.com/Zeroto521/PythonTemplate/releases/tag/v0.0.2

image

Comment (Zeroto521) [reply Flow.Launcher.Plugin.PythonTemplate/commands.py]: fixed, see https://github.com/Flow-Launcher/Flow.Launcher.Plugin.PythonTemplate/pull/8/commits/266cd0d0af8d45657e2cff4f7bed84f7a39b10f4

but this change may cause win env can't ignore those files or folders.

**Comment by Zeroto521** [comment]
ready to go

**Comment by jjw24** [comment]
@Zeroto521 can you approve + merge please

**Comment by jjw24** [comment]
approved

**Pull request merged**